[Support Us Page] Budget of Each Phase

CarsonKiibi opened the pull request:
**Budget of each phase component**
-simple transition animation
-mobile auto / play button works
-placeholder image
-layout changes from mobile/desktop as specified 
-using "primary" colour from tailwind which is different from figma color but they are both called primary so i assume the tailwind config just needs to be changed to that


<img width="808" alt="biomod-phases1v2" src="https://github.com/user-attachments/assets/32f60efc-9b3d-4606-a387-de2bab21a256">

<img width="220" alt="biomod-phases2" src="https://github.com/user-attachments/assets/80a8b2c9-ec6f-42c1-9757-217b7153198e">

CarsonKiibi committed:
-implemented MOBILE VIEW for the budget of each phase component
-basic transition animation
-manual navigation or play/pause automatic navigation (can switch between at any point)
-placeholder image
-easy to add new phases
-gonna update tomorrow (july 12) (it is july 12 rn just like 12:30am)

Commit:
-added desktop view

Commit:
-fixed text alignment issue

daniel-panhead commented on src/stories/supportus/PhaseBudget.stories.ts:
```diff
@@ -0,0 +1,18 @@
+import PhaseBudget from "@/components/supportus/PhaseBudget";
+
+const meta = {
+  title: "Support Us / Phase Budget",
```
Very nitpick but the spaces separating the slash aren't needed

daniel-panhead reviewed (changes requested):
Very very clean + polished :D

For the transitions I would recommend trying framer motion to make things simpler, although what you have right now works too so it's not required. I would also make the transition a bit shorter as right now it feels just a bit sluggish - maybe drop it to like 150ms? But otherwise this is stellar

IcedTea2K commented on src/components/supportus/PhaseBudget.tsx:
```diff
@@ -0,0 +1,215 @@
+  },
+];
+
+const PhaseBudget: React.FC = () => {
```
Nothing big, just for consistency (and preferences)
```suggestion
function PhaseBudget() {
```

CarsonKiibi replied:
The figma didnt have a button for that on desktop, would be easy to add but might clutter desktop view

IcedTea2K reviewed (changes requested):
Very nice!! Just wondering though. Does it also auto-play on laptop version (similar to on mobile?

Commit:
-changed title in .stories file
-changed transition speed
-changed component from arrow function to simple function

daniel-panhead reviewed (approved):
LGTM! pending @IcedTea2K
